output/csv/output.go

```diff
@@ -200,9 +204,17 @@ func MakeHeader(tags []string) []string {
 }
 
 // SampleToRow converts sample into array of strings
-func SampleToRow(sample *metrics.Sample, resTags []string, ignoredTags []string, row []string) []string {
+func SampleToRow(sample *metrics.Sample, resTags []string, ignoredTags []string, row []string,
+	timeFormat TimeFormat,
+) []string {
 	row[0] = sample.Metric.Name
-	row[1] = fmt.Sprintf("%d", sample.Time.Unix())
+
+	if timeFormat == RFC3399 {
+		row[1] = fmt.Sprint(sample.Time.Format(time.RFC3339))
+	} else {
+		row[1] = fmt.Sprintf("%d", sample.Time.Unix())
```

Comment:
```suggestion
		row[1] = strconv.Itoa(sample.Time.Unix())
```

Reply:
I tried this and the compiler complains `Itoa` expects `int` whereas `sample.Time.Unix()` is `int64`
```
cannot use sample.Time.Unix() (value of type int64) as int value in argument to strconv.Itoacompiler[IncompatibleAssign](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#IncompatibleAssign)
```